apps/drec-ui/src/api/organization.ts

```diff
@@ -30,3 +38,46 @@ export const useOrganization = () => {
         organizationLoading
     };
 };
+
+export const useMyOrganizationData = () => {
+    const { isLoading: organizationLoading, data: user } = useUserControllerMe();
+
+    const organization = user?.organization;
+    return { organizationLoading, organization };
+};
+
+export const useOrganizationMembersData = () => {
+    const { isLoading: isMembersLoading, data: members } =
+        useOrganizationControllerGetOrganizationUsersByCode();
```

Comment:
Why it is named by code if there is no code passed?
I understand that this is just a controller func naming, but still could be confusing

Reply:
Fixed this